WookieeMatt/Essence20
# Role points refactor; actor sheet UI

##### In this PR
- Role Points template and item sheet refactor
  - Tweaks for drops and prepare data to support this
- RP UI improvements on the actor sheet to show bonus and resource
  - Generating a bonus defense string and icon for this as well
- Adding an "Other" option for bonus type
- Error shown when dropping a RP directly onto an actor

##### Testing
- Make a bunch of different RPs, drop them on a Role, and drop that onto an actor
- Only when a starting value is provided for a bonus/resource will they appear on the actor sheet
- A `+` will appear for the bonus as long as the type isn't none or other
- Defense bonuses will show a letter for each applicable defense type
- Ensure RPs decrement their resource and personal power correctly, and an error shows up if you overspend
- Resting restores resources
- Dropping a RP onto an actor yields an error
